scripts/README.md

```diff
@@ -230,4 +230,71 @@ An `ERROR` can be shown if a duplicated `TAG` is detected in different `EVC`s. A
 Error: Detected duplicated 200 TAG in EVCs 861a11d8fce148 and d74e18464d524b in interface 00:00:00:00:00:00:00:01:1
 ```
 
+</details>
+
+<details><summary><h3>Add <code>special_available_tags</code> field to each Interface document in <code>interface_details</code> collection </h3></summary>
+
+[`special_vlan_allocation.py`](./special_vlan_allocation.py) is to add the new field ``special_available_tags`` to each interface document. This new field will keep track of special vlan usage:
```

Comment:
If you could also mention this script in the general information on changelog, that'd be great. Also did you have the chance to also execute and explore it? LGTM.

Reply:
Yes, I tested it. Although now it is outdated with the new variable `special_tag_range`.

Reply:
I also opened a new thread regarding the DB script if you could also address it and run it again, that'd be great https://github.com/kytos-ng/topology/pull/178#discussion_r1409317730, once you confirm the upgrade is working as intended it'll be shipped

Reply:
Excellent, let's close this thread then